### ISSUE TITLE
Entry point 'ngx-papaparse' contains deep imports using Angular 9

### ISSUE THREAD

**Issue by gitalvininfo** [alberthaff/ngx-papaparse#72]
**Describe the bug**

A clear and concise description of what the bug or problem is.
Everytime i run **ng serve** there is this warning that displays saying _Entry point 'ngx-papaparse' contains deep imports into_

**To Reproduce**

Please provide the necessary steps to reproduce the unwanted behaviour.
Previously, I added this package to an Angular 6 Project in stackblitz [here](https://stackblitz.com/edit/angular-download-upload-csv-6j2n5b) for testing purposes only.

Version of ngx-papaparse - 2.1.0
Version of Angular - 6.0.0

**Expected behavior**

I want to fixed this warning though it says that it is not a problem but I'm concern if this will continue to display when I run **ng serve** in the future, I'm currently in Angular 9.

**Software and platform**

 - OS: Linux
 - Browser: Chrome
 - Version: Version 84.0.4147.105 (Official Build) (64-bit)
 - Angular version: 9.0.7
 - Angular CLI version: 9.0.7
 - Node version: 12.19.0

**Additional context**

I've search several related topics about this, and the common answer is to ignore because it's not a major problem but I'm not satisfied of this.

Any clarification would be helpful. I'm not really sure if this thing must be taken seriously or not at all. Thanks.

**Comment by alberthaff** [comment]
Which version of ngx-papaparse are you using?
https://alberthaff.dk/projects/ngx-papaparse/docs/v5

You can choose between version in the upper left corner. Each version is only compatible with the stated Angular versions.

**Comment by gitalvininfo** [comment]
Im sorry I forgot, but I used the current ngx-papaparse right now which is 5.0.0 and Angular 9.0.7.

I don't actually know what is the cause for this issue.

**Comment by alberthaff** [comment]
Are you sure that you aren't using Angular 10?
Try reading this: https://alberthaff.dk/projects/ngx-papaparse/docs/v5/installation/angular-10-or-later

**Comment by gitalvininfo** [comment]
Im 300% sure that Im not using Angular 10 😕.
Yes Ive seen this solution which ignores the warning if you are using Angilar 10

This is my package.json for my version of angular
![Screenshot_20201015_185101_com.android.chrome~2.jpg](https://user-images.githubusercontent.com/14288144/96114071-b2997d00-0f17-11eb-9a79-a36d74b875e1.jpg)

Im confused as to why is this even happening since Im on Angular 9.

**Comment by alberthaff** [comment]
Please check your package.lock instead. The package.json file doesn't necessarily reflect what's installed.

AFAIK, the CommonJS error should not appear until NG10.

**Comment by gitalvininfo** [comment]
Closing this issue. My fault. 
Its actually one of my dev dependency that causes this warning which is **@angular-devkit/build-angular ^0.1001.6** . I just trace my previous version for this dev dependency then everything works fine.

In case of Ivy, I did not disable it and just leave it to default (which is enabled) and I think, this is nothing to do with Ivy IMO. 
I remember I run a command to clean cache for npm and hit npm install and this might cause the accidental update of my dev dependency.

Anyways thanks for telling me that package-lock.json must be checked first instead of the package.json.
